packages/larva-patterns/components/c-button/c-button.twig

```diff
@@ -1,5 +1,10 @@
 {% if c_button_url %}
-	<a class="c-button larva {{ modifier_class }} {{ c_button_classes }}" href="{{ c_button_url }}" target="{{ c_button_target_attr }}" rel="{{ c_button_rel_attr }}">
+	<a class="c-button larva {{ modifier_class }} {{ c_button_classes }}" href="{{ c_button_url }}" target="{{ c_button_target_attr }}" rel="{{ c_button_rel_attr }}"
+	<?php
```

Comment:
What is the proper way to inject php code like this to the twig template? Is there a twig command for do_action?

Reply:
Yep, anything ending in `{{ _wp_action }}` – can this whole block be in an action? e.g. only have

`{{ c_button_track_data_wp_action }}` and define the method `c_button_track_data` in the controller. PHP code in the Twig templates is not valid.